### PR TITLE
Recent glider cruises and glider vocab additions

### PR DESCRIPTION
**Pull request by cdobs**

### CODE REVIEW

**Comment by cdobs** [comment]
AT50-25

File: cruise/CruiseInformation.csv [hunk 1]
```diff
@@ -346,3 +346,6 @@ KH20240516,Kahuna,2024-05-16T09:00:00,2024-05-16T18:00:00,Pioneer glider cp_387
 KH20240523,Kahuna,2024-05-23T10:00:00,2024-05-23T20:00:00,Pioneer 20a glider deployments
 SKQ202408S,Sikuliaq,2024-05-28T00:00:00,2024-06-15T23:59:00,Papa 11
 AR84-01,Neil Armstrong,2024-05-31T00:00:00,2024-07-02T23:59:00,Irminger 11
+KH20240609,Kahuna,2024-06-09T09:00:00,2024-06-09T23:59:00,Pioneer glider cp_379 recovery
+AT50-25,Atlantis,2024-06-10T00:00:00,2024-06-15T23:59:00,Papa glider gp_515 recovery
+KH20240616,Kahuna,2024-06-16T09:00:00,2024-06-16T23:59:00,Pioneer glider cp_387 recovery
```
Comment (rtravis-whoi) [inline cruise/CruiseInformation.csv:351]: looks good

File: vocab/vocab.csv [hunk 2]
```diff
@@ -1718,6 +1718,14 @@ GI05MOAS-PG565-03-FLORTM000,Global Irminger Sea,Mobile Assets,Profiling Glider 5
...
+GI05MOAS-PG566,Global Irminger Sea,Mobile Assets,Profiling Glider 566,,,,0,200
+GI05MOAS-PG566-00-ENG000000,Global Irminger Sea,Mobile Assets,Profiling Glider 566,Mobile Asset Controller,Teledyne Webb,G2 Slocum Gliders,0,200
+GI05MOAS-PG566-01-CTDGVM000,Global Irminger Sea,Mobile Assets,Profiling Glider 566,CTD,Sea-Bird,SBE Glider Payload CTD (GP-CTD),0,200
+GI05MOAS-PG566-02-DOSTAM000,Global Irminger Sea,Mobile Assets,Profiling Glider 566,Dissolved Oxygen,Aanderaa,Optode 4831,0,200
+GI05MOAS-PG566-03-FLORTM000,Global Irminger Sea,Mobile Assets,Profiling Glider 566,3-Wavelength Fluorometer,WET Labs,ECO Puck FLBBCD-SLK,0,200
+GI05MOAS-PG566-04-FLORTO000,Global Irminger Sea,Mobile Assets,Profiling Glider 566,3-Wavelength Fluorometer,WET Labs,ECO BB3,0,200
+GI05MOAS-PG566-05-NUTNRM000,Global Irminger Sea,Mobile Assets,Profiling Glider 566,Nitrate,Satlantic,SUNA V2,0,200
+GI05MOAS-PG566-06-PARADM000,Global Irminger Sea,Mobile Assets,Profiling Glider 566,Photosynthetically Available Radiation,Biospherical Instruments,QSP-2155,0,200
```
Comment (snwhite) [inline vocab/vocab.csv:1728]: Looks good.

File: vocab/vocab.csv [hunk 3]
```diff
@@ -1880,6 +1888,14 @@ GP05MOAS-PG515-02-DOSTAM000,Global Station Papa,Mobile Assets,Profiling Glider 5
...
 GP05MOAS-PG515-05-NUTNRM000,Global Station Papa,Mobile Assets,Profiling Glider 515,Nitrate,Satlantic,SUNA V2,0,200
+GP05MOAS-PG563,Global Station Papa,Mobile Assets,Profiling Glider 563,,,,0,200
+GP05MOAS-PG563-00-ENG000000,Global Station Papa,Mobile Assets,Profiling Glider 563,Mobile Asset Controller,Teledyne Webb,G2 Slocum Gliders,0,200
+GP05MOAS-PG563-01-CTDGVM000,Global Station Papa,Mobile Assets,Profiling Glider 563,CTD,Sea-Bird,SBE Glider Payload CTD (GP-CTD),0,200
+GP05MOAS-PG563-02-DOSTAM000,Global Station Papa,Mobile Assets,Profiling Glider 563,Dissolved Oxygen,Aanderaa,Optode 4831,0,200
+GP05MOAS-PG563-03-FLORTM000,Global Station Papa,Mobile Assets,Profiling Glider 563,3-Wavelength Fluorometer,WET Labs,ECO Puck FLBBCD-SLK,0,200
+GP05MOAS-PG563-04-FLORTO000,Global Station Papa,Mobile Assets,Profiling Glider 563,3-Wavelength Fluorometer,WET Labs,ECO BB3,0,200
+GP05MOAS-PG563-05-NUTNRM000,Global Station Papa,Mobile Assets,Profiling Glider 563,Nitrate,Satlantic,SUNA V2,0,200
+GP05MOAS-PG563-06-PARADM000,Global Station Papa,Mobile Assets,Profiling Glider 563,Photosynthetically Available Radiation,Biospherical Instruments,QSP-2155,0,200
```
Comment (snwhite) [inline vocab/vocab.csv:1898]: Looks good.

File: cruise/CruiseInformation.csv (hunk 1 above)
Comment (snwhite) [inline cruise/CruiseInformation.csv:351]: Looks good.

**Review by snwhite — APPROVED** [review]

File: vocab/vocab.csv (hunk 2 above)
Comment (rtravis-whoi) [inline vocab/vocab.csv:1728]: looks good

File: vocab/vocab.csv (hunk 3 above)
Comment (rtravis-whoi) [inline vocab/vocab.csv:1898]: looks good

**Review by rtravis-whoi — APPROVED** [review]

**Pull request merged**